plop-templates/Component/Component.tsx.hbs

```diff
@@ -12,7 +12,7 @@ export interface Props {
 /**
  * BETA: This component is still a work in progress and is subject to change.
  *
- * `import {{inBraces name}} from "@chanzuckerberg/eds";`
+ * `import {{inBraces (pascalCase name)}} from "@chanzuckerberg/eds";`
```

Comment:
I removed the dep on pascal-case rather than update it.

Here we're swapping it out with the built in HandleBars `pascalCase`. Tested out by running `yarn plop`.